style.css

```diff
@@ -97,10 +342,22 @@ a:active {
   text-decoration: none;
 }
 
+.button-link {
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #396df2;
+  width: 162px;
+  height: 48px;
+}
+
```

Comment:
- [ ] Please make sure you applied the same font and size for all the buttons as the Figma design.
# your design 👇
![seeproject](https://user-images.githubusercontent.com/66992901/191830744-7bdbd6eb-335e-41ef-b239-e227efab41e6.PNG)
# Figma desing👇
![seeprojectfigma](https://user-images.githubusercontent.com/66992901/191830778-4b1ccc19-c95c-4a59-a62e-2a11647f4607.PNG)